packages/node_modules/@webex/plugin-rooms/test/integration/spec/rooms.js

```diff
@@ -196,6 +198,89 @@ describe('plugin-rooms', function () {
       });
     });
 
+    describe('#listWithReadStatus()', () => {
+      // Reminder: we can't run the next two creates in parallel because some of
```

Comment:
The tests should be independent so that we can isolate and skip tests to troubleshoot.

Reply:
Updated for independent tests.